### PR TITLE
Removed updateConstraintsIfNeeded from ARNavigationButton

### PR DESCRIPTION
**Pull request by sarahscott**
Does anyone foresee any repercussions for removing this line?

### CODE REVIEW

File: CHANGELOG.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ upcoming:
   details: Analytics improvements
 
   dev:
+    - Removed updateConstraintsIfNeeded from ARNavigationButton - sarah
```
Comment (alloy) [inline CHANGELOG.yml]: The changelog should describe what issue was solved. Readers that might know the bug’s effect will not necessarily understand that this change solves that issue.

**Comment by alloy** [comment]
According to @orta this line only existed for tests.

Can you rebase please?

**Comment by sarahscott** [comment]
🍏

**Comment by alloy** [comment]
👍

**Pull request merged**